context-monitoring/src/main/java/de/atb/context/monitoring/monitors/file/FilePairSystemMonitor.java

```diff
@@ -398,38 +152,4 @@ protected final void handleFile(final String fileName, final Long time,
         }
         return this.filePair;
```

Comment:
`updateFilePair` can be void because it only modifies `this.filePair` and the return value is never used

Reply:
[x] done